specification/src/main/asciidoc/platform/PlatformOverview.adoc

```diff
@@ -1073,3 +1073,7 @@ SE 8 and to extend the range of validated objects. While the focus of
 CDI in this release is to extend its scope beyond Java EE with the
 introduction of a bootstrapping API, CDI also includes enhancements for
 event processing and alignment on Java SE 8 features.
+
```

Comment:
I kept the changelog for the old Java EE untouched. Do we want to remove it?

Reply:
Yes.

Reply:
Leave as is.  We may remove it in Jakarta EE 9, but not now.